kas-installer-defaults.env

```diff
@@ -142,7 +137,18 @@ KAS_FLEET_MANAGER_GIT_URL="${KAS_FLEET_MANAGER_GIT_URL:-"https://github.com/bf2f
 # KAS Fleet Manager's git reference. A commit ID, branch name or tag can be used. The commit ID should be compatible with
 # the container image contents used
 #
-KAS_FLEET_MANAGER_GIT_REF="${KAS_FLEET_MANAGER_GIT_REF:-"752566e48829300669a3512c2af4236ba622a4e8"}"
+KAS_FLEET_MANAGER_GIT_REF="${KAS_FLEET_MANAGER_GIT_REF:-"7f968f0b39ffbb509d55fee01bea8b06c85324bc"}"
+
+# [optional]
+# KAS Fleet Manager container image tag. Defaults to first seven characters of the KAS_FLEET_MANAGER_GIT_REF
+#
+KAS_FLEET_MANAGER_IMAGE_TAG="${KAS_FLEET_MANAGER_IMAGE_TAG:-${KAS_FLEET_MANAGER_GIT_REF:0:7}}"
+
+# [optional]
+# Enable enterprise cluster registration. Set this value to true in kas-installer.env or the environment when the
+# installation is intended for use with dynamically registered enterprise data plane clusters.
+#
+ENTERPRISE_ENABLED="${ENTERPRISE_ENABLED:-"false"}"
```

Comment:
Not sure if a simple boolean is enough. Should we have this flag/option support a mixed enterprise+managed mode also?

Reply:
> Not sure if a simple boolean is enough.

No strong opinion from end. But I'll like to mention a point below.

> Should we have this flag/option support a mixed enterprise+managed mode also?

If we take this route, we've to be clear that the same data plane cluster cannot be used as an enterprise cluster and a managed cluster at the same time.

Reply:
> If we take this route, we've to be clear that the same data plane cluster cannot be used as an enterprise cluster and a managed cluster at the same time.

That's true. Do you think it would be worth having KFM validate that when handling cluster registration requests? It's probably not a critical feature at this point. I'm fine with leaving this as true/false for now.